superset-frontend/plugins/plugin-chart-echarts/src/Heatmap/transformProps.ts

```diff
@@ -119,10 +123,11 @@ export default function transformProps(
 
   let [min, max] = (valueBounds || []).map(parseAxisBound);
   if (min === undefined) {
-    min = minBy(data, row => row[colorColumn])?.[colorColumn] as number;
+    min = (minBy(data, row => row[colorColumn])?.[colorColumn] as number) || 0;
   }
   if (max === undefined) {
-    max = maxBy(data, row => row[colorColumn])?.[colorColumn] as number;
+    max =
+      (maxBy(data, row => row[colorColumn])?.[colorColumn] as number) || 200;
```

Comment:
Where is the value of `200` coming from?

Reply:
This is coming from the [default values](https://echarts.apache.org/en/option.html#visualMap-continuous.max) of ECharts. I added a constant to better describe the value.